WPMedia/arc-themes-components
# Themes 1480: Enable tests (with reduced coverage constraints for now)

## Ticket

- [THEMES-1480](https://arcpublishing.atlassian.net/browse/THEMES-1480)

## Description

Re-enable testing

## Test Steps

1. Checkout branch - `git checkout themes-1480`
2. Update dependencies - `npm i`
3. Run Storybook `npm run test -- --coverage`
4. Ensure tests run and pass new coverage

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._


## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._



[THEMES-1480]: https://arcpublishing.atlassian.net/browse/THEMES-1480?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
